src/js/components/RangeInput/RangeInput.js

```diff
@@ -1,40 +1,119 @@
 import React, { forwardRef, useContext, useState } from 'react';
-
+import { Box } from '../Box';
+import { TextInput } from '../TextInput';
 import { FormContext } from '../Form/FormContext';
 import { StyledRangeInput } from './StyledRangeInput';
 
+const allowedKeysToOpenInputValue = [
+  'ArrowRight',
+  'ArrowLeft',
+  'ArrowUp',
+  'ArrowDown',
+];
+const allowedKeysToCloseInputValue = ['Tab', 'Escape'];
+
 const RangeInput = forwardRef(
   (
-    { a11yTitle, name, onChange, onFocus, onBlur, value: valueProp, ...rest },
+    {
+      a11yTitle,
+      name,
+      onChange,
+      onFocus,
+      onBlur,
+      onKeyDown,
+      min = 0,
+      max = 100,
+      step = 1,
+      inputValue = false,
```

Comment:
Do we need to hardcode these values ?
`min` `max` `step`

Reply:
Yes, because if these values are not entered, the component will assume the default values